scripts/variantstore/wdl/extract/gvs_vds_tie_out.py

```diff
@@ -0,0 +1,201 @@
+# gvs_vds_tie_out.py
+
+import hail as hl
+import pytest
+
+mt = hl.read_matrix_table('joined.mt')
+rs = mt.rows()
+cs = mt.cols()
+es = mt.entries()
+
+
+"""
+Schema of `mt`:
+----------------------------------------
+Global fields:
+    'tranche_data': array<struct {
+        model: str,
+        truth_sensitivity: float64,
+        min_vqslod: float64,
+        filter_name: str
+    }>
+    'truth_sensitivity_snp_threshold': float64
+    'truth_sensitivity_indel_threshold': float64
+    'snp_vqslod_threshold': float64
+    'indel_vqslod_threshold': float64
+----------------------------------------
+Column fields:
+    's': str
+    'vcf_col': struct {
+        s: str
+    }
+    'vds_col': struct {
+        s: str
+    }
+----------------------------------------
+Row fields:
+    'locus': locus<GRCh38>
+    'vcf_row': struct {
+        locus: locus<GRCh38>,
+        alleles: array<str>,
+        rsid: str,
+        qual: float64,
+        filters: set<str>,
+        info: struct {
+            AC: array<int32>,
+            AF: array<float64>,
+            AN: int32,
+            AS_QUALapprox: str,
+            AS_VQSLOD: array<str>,
+            AS_YNG: array<str>,
+            QUALapprox: int32
+        }
+    }
+    'vds_row': struct {
+        locus: locus<GRCh38>,
+        alleles: array<str>,
+        filters: set<str>,
+        as_vqsr: dict<str, struct {
+            model: str,
+            vqslod: float64,
+            yng_status: str
+        }>
+    }
+----------------------------------------
+Entry fields:
+    'vcf_entry': struct {
+        AD: array<int32>,
+        FT: str,
+        GQ: int32,
+        GT: call,
+        RGQ: int32
+    }
+    'vds_entry': struct {
+        LGT: call,
```

Comment:
Dont we also have a GT in the VDS? isn't that what messed us up? or do we need to drop it?

Reply:
Actually this appears to be correct as is.